directjngine/src/main/java/com/softwarementors/extjs/djn/router/processor/standard/form/upload/FileUploadException.java

```diff
@@ -37,10 +37,11 @@ protected FileUploadException(String message, Throwable cause) {
   public static FileUploadException forFileUploadException( org.apache.commons.fileupload.FileUploadException cause ) {
     assert cause != null;
     
-    // The FileUploadException raised by commons-fileupload always
-    // hides an inner exception that is the real cause
+    // The FileUploadException raised by commons-fileupload usually hides an inner exception that is the real cause
     Throwable realCause = cause.getCause();
-    assert realCause != null;
+    if (realCause == null) {
```

Comment:
So there wasn't a "realCause" here (though the message would seem to indicate that there should have been). If that happens, rather than just throwing a blind NPE we should just use the original (safe) cause for the message.